package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@browserstack/wdio-browserstack-service",
+  "version": "1.0.0",
+  "description": "WebdriverIO service for better Browserstack integration",
+  "author": "Browserstack",
+  "homepage": "https://github.com/browserstack/wdio-browserstack-service",
+  "license": "MIT",
+  "main": "./build",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/browserstack/wdio-browserstack-service.git"
+  },
+  "keywords": [
+    "webdriverio",
+    "wdio",
+    "browserstack",
+    "wdio-service"
+  ],
+  "bugs": {
+    "url": "https://github.com/browserstack/wdio-browserstack-service/issues"
+  },
+  "dependencies": {
+    "@types/node": "^17.0.4",
+    "@wdio/logger": "7.16.0",
+    "@wdio/types": "7.16.14",
+    "browserstack-local": "^1.4.5",
+    "got": "^11.0.2",
+    "webdriverio": "7.16.15"
+  },
+  "devDependencies": {
+    "@tsconfig/node12": "^1.0.9"
```

Comment:
Why is node12 in devDependencies? If we are using this config in our code, then we should include it in our dependencies else when npm release the package and user tries to install it using `npm i` command, this package doesn't get installed on it's own.

Reply:
jest seems to be missing in devDependencies

Reply:
yes, will add jest needed for tests